### PR TITLE
Blog Post: Harry, an Open Source Fuzz Testing and Verification Tool for Apache Cassandra.

### PR DESCRIPTION
**Pull request by ifesdjeen**

### CODE REVIEW

**Comment by nonstopdtop** [comment]
https://issues.apache.org/jira/browse/CASSANDRA-17066

File: site-content/source/modules/ROOT/pages/blog.adoc [hunk 1]
```diff
@@ -1,762 +1,812 @@
...
+:page-layout: blog-landing
+:page-role: blog-landing
+
+////
+NOTES FOR CONTENT CREATORS
+- To add a new blog post, copy and paste markup for one card below.  Copy from '//start' to the next '//end'
+- Replace post tile, date, description and link to you post.
+////
+
+//start card
```
Comment (ErickRamirezDS) [inline site-content/source/modules/ROOT/pages/blog.adoc]: Please update your commit so only lines 17-34 are modified on `blog.adoc`. Thanks! 🍻

File: site-content/source/modules/ROOT/pages/blog/Harry-an-Open-Source-Fuzz-Testing-and-Verification-Tool-for-Apache-Cassandra.adoc [hunk 2]
```diff
@@ -0,0 +1,523 @@
+= Harry, an Open Source Fuzz Testing and Verification Tool for Apache Cassandra
+:page-layout: single-post
+:page-role: blog-post
+:page-post-date: November 2, 2021
+:page-post-author: The Apache Cassandra Community
```
Comment (ErickRamirezDS) [inline site-content/source/modules/ROOT/pages/blog/Harry-an-Open-Source-Fuzz-Testing-and-Verification-Tool-for-Apache-Cassandra.adoc]: Please update the author to `Alex Petrov`. 🍻

File: site-content/source/modules/ROOT/pages/blog/Harry-an-Open-Source-Fuzz-Testing-and-Verification-Tool-for-Apache-Cassandra.adoc [hunk 3]
```diff
@@ -0,0 +1,523 @@
...
+
+== Fuzz testing 
+
+Since most of the bugs are reproduced by taking a sequence of actions
+following some pattern, we need to specify what actions can be used to
+lead to a given state. However, there’s a lot of flexibility regarding
+which values exactly are written in specific columns.
+
+For example, if we look at
+https://issues.apache.org/jira/browse/CASSANDRA-16453?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel[CASSANDRA-16453,window=_blank],
```
Comment (ErickRamirezDS) [inline site-content/source/modules/ROOT/pages/blog/Harry-an-Open-Source-Fuzz-Testing-and-Verification-Tool-for-Apache-Cassandra.adoc]: Please clean up the link to just use `https://issues.apache.org/jira/browse/CASSANDRA-16453`. 🙂

File: site-content/source/modules/ROOT/pages/blog/Harry-an-Open-Source-Fuzz-Testing-and-Verification-Tool-for-Apache-Cassandra.adoc [hunk 4]
```diff
@@ -0,0 +1,523 @@
...
+key. Each slice is then inflated using a bijection that corresponds to
+the part of the key it represents. To convert the value back to the
+descriptor, we must deflate each part of the key and then “stitch” the
+values back together into a 64-bit descriptor.
+
+To summarise, key generators are just bijections that can generate
+multiple values for a single 64-bit descriptor instead of one. A
+simplified and generalized version of such bijection may look
+something like this:
+
```
Comment (ErickRamirezDS) [inline site-content/source/modules/ROOT/pages/blog/Harry-an-Open-Source-Fuzz-Testing-and-Verification-Tool-for-Apache-Cassandra.adoc:399]: For consistency, insert the line `[source,java]` here.

File: site-content/source/modules/ROOT/pages/blog/Harry-an-Open-Source-Fuzz-Testing-and-Verification-Tool-for-Apache-Cassandra.adoc [hunk 5]
```diff
@@ -0,0 +1,523 @@
...
+starting with how to reproduce a sequence of operations up to how the
+values are generated. Using this information, we can create a
+quiescent model checker that can validate the state of the database in
+the absence of in-flight operations, assuming we know the state of all
+operations before this moment.
+
+As we’ve discussed, Harry is working with reproducible histories of
+operations, where the following information identifies each operation:
+
+----
```
Comment (ErickRamirezDS) [inline site-content/source/modules/ROOT/pages/blog/Harry-an-Open-Source-Fuzz-Testing-and-Verification-Tool-for-Apache-Cassandra.adoc:438]: For consistency, insert the line `[source,java]` here.

File: site-content/source/modules/ROOT/pages/blog/Harry-an-Open-Source-Fuzz-Testing-and-Verification-Tool-for-Apache-Cassandra.adoc [hunk 6]
```diff
@@ -0,0 +1,523 @@
...
+  long lts; // logical timestamp of the operation
+  long pd;  // partition descriptor, derived from LTS
+  long cd;  // clustering descriptor, derived from LTS and PD
+  long OperationKind; // operation type, derived from LTS and PD
+}
+----
+
+Now, all we need to do is to produce a sequence of operations. For example, each operation with `INSERT` kind is going to be represented by:
+
+----
```
Comment (ErickRamirezDS) [inline site-content/source/modules/ROOT/pages/blog/Harry-an-Open-Source-Fuzz-Testing-and-Verification-Tool-for-Apache-Cassandra.adoc:450]: For consistency, insert the line `[source,java]` here.

**Review by ErickRamirezDS — CHANGES_REQUESTED** [review]

**Comment by ifesdjeen** [comment]
@ErickRamirezDS fixed all your suggestions

**Comment by ErickRamirezAU** [comment]
Sorry. I somehow missed the notification and didn't see your update. Staging now...

**Review by ErickRamirezAU — APPROVED** [review]
Ready to commit 👍

**Comment by michaelsembwever** [comment]
manually merged with https://github.com/apache/cassandra-website/commit/e35418ad9542cc50f61e9bcb3610a5a457657bd3

**Pull request closed without merging**